src/System.Transactions/src/System/Transactions/TemporaryStubs.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+
+// TODO #9582: Replace with real IFormatter / BinaryFormatter when available
+
+namespace System.Runtime.Serialization
+{
+    internal interface IFormatter
+    {
+        object Deserialize(Stream serializationStream);
+        void Serialize(Stream serializationStream, object graph);
+    }
+}
+
+namespace System.Runtime.Serialization.Formatters.Binary
+{
+    internal class BinaryFormatter : IFormatter
```

Comment:
I see that you have another PR out for the BinaryFormatter. So I assume we can make the adjustment once that's merged.

---
In reply to: [71207092](https://github.com/dotnet/corefx/pull/10089#discussion_r71207092) [](ancestors = 71207092)